package.json

```diff
@@ -11,6 +11,7 @@
     "node": ">=0.8"
   },
   "dependencies": {
+    "codecov.io": "0.1.6",
```

Comment:
Is it a good idea to use an exact version here? It seems to be updated several times a year..